lib/mocks/billdispenser.js

```diff
@@ -1,17 +1,17 @@
-const _ = require('lodash')
-const Bluebird = require('bluebird')
+const _ = require('lodash/fp')
+const pDelay = require('delay')
 
 var BillDispenser = function (config) {
   this.initialized = false
   this.initializing = false
   this.config = config
-  this.matchMockedErrors = _.wrap(_.get(this.config, 'mockedError', {}), (errors, notes) => {
+  this.matchMockedErrors = _.wrap((errors, notes) => {
     const p = _.toPairs(errors)
     const f = _.conforms({
       1: (v) => _.isEqual(v, notes)
     })
-    return _.get(_.filter(p, f), [0, 0])
-  })
+    return _.get([0, 0], _.filter(f, p))
+  }, _.get('mockedError', this.config, {}))
```

Comment:
Looks better but this is pretty complex code. Have you tested it? Should be easy to unit test since it's pretty functional.

Reply:
done